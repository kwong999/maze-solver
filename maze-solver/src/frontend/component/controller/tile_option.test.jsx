import React from 'react';
import TileOption from './tile_option';
import renderer from 'react-test-renderer';

test('TileOption with option "TEST"', () => {
  const component = renderer.create(
    <TileOption
      currentTileType={"TEST"}
      tileType={"TEST"}
      changeTileType={() => { }}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})