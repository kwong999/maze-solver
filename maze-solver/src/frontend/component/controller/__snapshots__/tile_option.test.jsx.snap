// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TileOption with option "TEST" 1`] = `
<div
  className="tile-option"
>
  <div>
    <p>
      Current Tile Type: 
      <span>
        TEST
      </span>
    </p>
    <div
      className="TEST"
    />
  </div>
  <button>
    <div>
      <p>
        Blank
      </p>
      <div
        className="blank"
      />
    </div>
  </button>
  <button>
    <div>
      <p>
        Wall
      </p>
      <div
        className="wall"
      />
    </div>
  </button>
  <button>
    <div>
      <p>
        Start
      </p>
      <div
        className="start"
      />
    </div>
  </button>
  <button>
    <div>
      <p>
        End
      </p>
      <div
        className="end"
      />
    </div>
  </button>
  <p>
    Left click to update tile.
  </p>
</div>
`;
